docs/pages/access-controls/sso/google-workspace.mdx

```diff
@@ -356,50 +423,17 @@ Create the connector using the `tctl` tool:
 $ tctl create -f gworkspace-connector.yaml
 ```
 
-<Details title="Upgrading from Teleport < 8.1.2?">
```

Comment:
FWIW I just [recently](https://github.com/gravitational/teleport/pull/28117) moved this content into the details block to reduce guide length while preserving it for any stragglers. 

> Remove most references to issues from Teleport pre 8.1, left one in a details block.